build.gradle

```diff
@@ -157,7 +157,7 @@ publishing {
 
 dependencies {
     implementation "org.eclipse.lsp4j:org.eclipse.lsp4j:0.14.0"
-    implementation "software.amazon.smithy:smithy-model:[1.30.0, 2.0["
+    implementation "software.amazon.smithy:smithy-model:[1.31.0, 2.0["
```

Comment:
hey @srchase I'm wondering what this syntax means: `[1.31.0, 2.0[` - I understand `[1.31.0` means we need _at least 1.31.0 including that version_, but the `2.0[` part puzzles me. Any hints?

Reply:
You got it. That syntax is a little funky.